bitnami/thanos/templates/query/deployment.yaml

```diff
@@ -95,10 +95,16 @@ spec:
             {{- end }}
```

Comment:
Some lines above (I can't mark them here since they're out of the scope of the PR), there is the same logic but applicable to the server, do you think the modular approach should be applied to the server as well?

Reply:
Indeed, this make sense for `--grpc-server-tls-client-ca` option, I've pushed a change to apply the same logic.